Specifying options.source in .neutrinorc.js doesn't seem to work

In this mission control project (https://github.com/mozilla/missioncontrol) I want to move the frontend code into a directory called 'frontend/' (to distinguish it from the django portions). However, when I try to do this I get this error:

```
Hammersmith:missioncontrol-test wlach$ yarn start
yarn start v0.21.3
$ neutrino start 
✔ Development server running on: http://localhost:5000
✔ Build completed

ERROR in multi ./~/react-hot-loader/patch.js (webpack)-dev-server/client?http://localhost:5000 (webpack)/hot/dev-server.js ./src/index
Module not found: Error: Can't resolve '/Users/wlach/src/missioncontrol-test/src/index' in '/Users/wlach/src/missioncontrol-test'
resolve '/Users/wlach/src/missioncontrol-test/src/index' in '/Users/wlach/src/missioncontrol-test'
  using description file: /Users/wlach/src/missioncontrol-test/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/wlach/src/missioncontrol-test/package.json (relative path: .)
    using description file: /Users/wlach/src/missioncontrol-test/package.json (relative path: ./src/index)
      as directory
        /Users/wlach/src/missioncontrol-test/src/index doesn't exist
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /Users/wlach/src/missioncontrol-test/src/index doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/wlach/src/missioncontrol-test/src/index.js doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /Users/wlach/src/missioncontrol-test/src/index.json doesn't exist
      .jsx
        Field 'browser' doesn't contain a valid alias configuration
        /Users/wlach/src/missioncontrol-test/src/index.jsx doesn't exist
[/Users/wlach/src/missioncontrol-test/src/index]
[/Users/wlach/src/missioncontrol-test/src/index]
[/Users/wlach/src/missioncontrol-test/src/index.js]
[/Users/wlach/src/missioncontrol-test/src/index.json]
[/Users/wlach/src/missioncontrol-test/src/index.jsx]
 @ multi ./~/react-hot-loader/patch.js (webpack)-dev-server/client?http://localhost:5000 (webpack)/hot/dev-server.js ./src/index
```

This is my modified .neutrinorc.js:

```js
module.exports = {
  options: {
    source: 'frontend'
  },
  use: [
    'neutrino-preset-mozilla-rpweb',
    'neutrino-preset-jest'
  ]
};
```

--- a/packages/neutrino/src/api.js
+++ b/packages/neutrino/src/api.js
@@ -55,6 +55,29 @@ const getOptions = (opts = {}) => {
   return options;
 };
 
+/* eslint-disable no-param-reassign */
+const mergeOptions = (options, newOptions) => {
+  const paths = pathOptions.map(([path]) => path);
+
+  Object
+    .keys(newOptions)
+    .forEach((key) => {
+      if (paths.includes(key)) {
+        options[key] = newOptions[key];
+        return;
+      }
+
+      if (!options[key]) {
+        options[key] = newOptions[key];
+      } else {
+        options[key] = merge(options[key], newOptions[key]);
+      }
+    });
+
+  return options;
+};
+/* eslint-enable no-param-reassign */
+
 // Api :: Object? -> Object
 const Api = pipe(getOptions, (options) => {
   const listeners = {};
@@ -104,7 +127,7 @@ const Api = pipe(getOptions, (options) => {
       // options
       [is(Object), () => {
         if (middleware.options) {
-          api.options = getOptions(merge(api.options, middleware.options));
+          api.options = mergeOptions(api.options, middleware.options);
         }
 
         if (middleware.env) {
@@ -122,7 +145,7 @@ const Api = pipe(getOptions, (options) => {
                 return env;
               }
 
-              api.options = getOptions(merge(api.options, env.options));
+              api.options = mergeOptions(api.options, env.options);
 
               return omit(['options'], env);
             });

